tests/testdata.nix

```diff
@@ -33,6 +33,10 @@ let
 
         touches 10-subdir-ignoring-itself/foo {foo,bar}
         echo foo >10-subdir-ignoring-itself/foo/.gitignore
+
+        mkdir 12-empty-dir
+        mkdir 12-not-empty-dir
+        touch 12-not-empty-dir/just-a-regular-file
```

Comment:
Where does it assert that it wasn't included?

Reply:
These tests assert that the `git` behavior matches the gitignore Nix behavior.

Reply:
https://github.com/hercules-ci/gitignore/blob/master/tests/runner.nix#L33